lib/dal/src/func/authoring.rs

```diff
@@ -116,7 +132,26 @@ impl FuncAuthoringClient {
         create::create_func(ctx, kind, name, options).await
     }
 
-    /// Saves a [`Func`] and returns the [result](SavedFunc).
+    /// Performs a "dummy" [`Func`] execution and returns the [result](DummyExecutionResult).
+    #[instrument(name = "func.authoring.dummy_execute_func", level = "info", skip_all)]
+    pub async fn dummy_execute_func(
```

Comment:
Next pass: make it clear in the SDF routes that the `execute` route is for _test_ executions and has nothing to do with `save_and_exec`.

We'll also align all the naming to use either "dummy" or "test", but probably "test".